src/components/ActivityCard.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import styled from 'styled-components'
+import moment from 'moment'
+import 'moment/locale/de'
+moment.locale('de')
+
+const activityTypeIcon = {
+  Ride: '/assets/bike.svg',
+  Workout: '/assets/dumbbell.svg',
+  WeightTraining: '/assets/dumbbell.svg',
+  Run: '/assets/run.svg'
+}
+
+const StyledCard = styled.article`
+  background: var(--grey);
+  border-radius: 10px;
+  padding: 10px 15px;
+  margin-bottom: 20px;
+`
+
+const StyledCardHeader = styled.div`
+  align-items: center;
+  border-bottom: 1px solid var(--dark-font);
+  display: flex;
+  height: 30px;
+  padding: 0 10px 5px;
+  justify-content: space-between;
+
+  h2 {
+    font-size: 0.7em;
+  }
+
+  p {
+    font-size: 0.7em;
+  }
+`
+
+const StyledCardBody = styled.div`
```

Comment:
You can think about using a `section` instead of an unspecific `div`

Reply:
corrected :)